ui/react-components/components/ColorPicker/ColorPicker.jsx

```diff
@@ -0,0 +1,81 @@
+import React, { useState, useEffect, useRef } from "react";
+import classNames from "classnames";
+import {
+  colorPickerWrapper,
+  colorBoxTrigger,
+  colorBox
+} from "./ColorPicker.module.scss";
+import PropTypes from "prop-types";
+
+function ColorPicker(props) {
+  const [showColorPicker, setShowColorPicker] = useState(false);
+  const [selectedColor, setSelectedColor] = useState(props.selectedColor || "");
+  const ref = useRef(null);
+
+  const handleClickOutside = event => {
+    if (ref.current && !ref.current.contains(event.target)) {
+      setShowColorPicker(false);
+    }
+  };
+
+  useEffect(() => {
+    document.addEventListener("click", handleClickOutside, true);
+    return () => {
+      document.removeEventListener("click", handleClickOutside, true);
+    };
+  }, []);
+
+  const toggleColorPicker = () => {
+    setShowColorPicker(!showColorPicker);
+  };
```

Comment:
You can use just one line. No need of curly braces.
`const toggleColorPicker = () => setShowColorPicker(!showColorPicker);`

Reply:
Done